py/client-ticking/setup.py

```diff
@@ -0,0 +1,48 @@
+#
+#  Copyright (c) 2016-2021 Deephaven Data Labs and Patent Pending
+#
+import pathlib
+from setuptools import find_packages, setup, Extension
+from Cython.Build import cythonize
+
+# The directory containing this file
+HERE = pathlib.Path(__file__).parent
+
+# The text of the README file
+README = (HERE / "README.md").read_text()
+
+setup(
+    name='pydeephaven-ticking',
+    version='0.26.0',
```

Comment:
```suggestion
    version='0.26.0.dev',
```

Reply:
Looks like is a manual update when we release, just like it is for the existing client:
https://github.com/deephaven/deephaven-core/blob/bbd9d23dfa86da75498545588edde0a960dcbb47/py/client/setup.py#L15
https://github.com/deephaven/deephaven-core/blob/bbd9d23dfa86da75498545588edde0a960dcbb47/RELEASE.md#11-deephaven-python-client
I believe Jianfeng updates these manually.